sbin/build.sh

```diff
@@ -478,9 +478,9 @@ makeACopyOfLibFreeFontForMacOSX() {
 getFirstTagFromOpenJDKGitRepo()
 {
     git fetch --tags "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}"
-    justOneFromTheRevList=$(git rev-list --tags --max-count=1)
-    tagNameFromRepo=$(git describe --tags "$justOneFromTheRevList")
-    echo "$tagNameFromRepo"
+    RevList=$(git rev-list --tags --max-count=100)
```

Comment:
style issue with this var name

Reply:
Not sure on the performance impact of pulling down 100

Reply:
100 is a magic number

Reply:
When I tried it on the command line is was close enough to instantaneous not to be a concern